pkg/apis/imageregistry/v1alpha1/imageregistry_types.go

```diff
@@ -23,6 +27,13 @@ type ImageRegistrySpec struct {
 	PlacementRef PlacementRef `json:"placementRef"`
 }
 
+type Registries struct {
+	// Mirror is the mirrored registry of the source
+	Mirror string `json:"mirror"`
+	// Source is the source registry
+	Source string `json:"source"`
```

Comment:
this is optional?

Reply:
yes. add doc for this filed.

Reply:
what will be the source when this value is empty?

Reply:
as the same as using `Registry` only. the registry address path of all images will be replaced by the Mirror.